carml-engine/src/test/resources/RmlMapper/nullValues/nullValueTemplate.rml.ttl

```diff
@@ -0,0 +1,18 @@
+@prefix rr: <http://www.w3.org/ns/r2rml#>.
+@prefix rml: <http://semweb.mmlab.be/ns/rml#>.
+@prefix ex: <http://example.com/>.
+@prefix ql: <http://semweb.mmlab.be/ns/ql#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+<#TermTypeMappingLiteral> a rr:TriplesMap ;
+  rml:logicalSource [
+    rml:source "nullValues.json" ;
+    rml:referenceFormulation ql:JSONPath ;
+    rml:iterator "$.Child[*]" ;
+  ] ;
+
+  rr:subjectMap [
+    rr:template "http://example.com/Child/{first}{last}" ;
```

Comment:
`first` is `null`, so no triples are generated, correct?

Reply:
correct